src/main/java/core/basesyntax/ColorSuplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSuplier {
+    public static Color getRandomColor() {
+        int colorIndex = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[colorIndex];
+        return color;
```

Comment:
```suggestion
        return Color.values()[colorIndex];;
```
Usually, IDE highlights such moments, doesn't your one do it?